oid4vci/oid4vci/public_routes.py

```diff
@@ -297,6 +298,31 @@ class IssueCredentialRequestSchema(OpenAPISchema):
     proof = fields.Dict(metadata={"description": ""})
 
 
+class ICredProcessor(ABC):
+    """Returns singed credential payload."""
+
+    @abstractmethod
+    def issue_cred(
+        self,
+        body: any,
+        supported: SupportedCredential,
+        ex_record: OID4VCIExchangeRecord,
+        pop: PopResult,
+        context: AdminRequestContext,
+    ):
+        """Method signature.
+
+        Args:
+            body: any
+            supported: SupportedCredential
+            ex_record: OID4VCIExchangeRecord
+            pop: PopResult
+            context: AdminRequestContext
+        Returns:
+            encoded: signed credential payload.
+        """
+
+
```

Comment:
I think this class ought to be defined in a location that is better suited for import from cred processor plugins. I'd consider putting it in the root of the oid4vci package.

Reply:
I encountered a circular import issue after moving the code to a separate file cred_processor.py, which imports PopResult from public_routes.py. However, public_routes.py also requires ICredProcessor from cred_processor.py.

Are you okay with moving PopResult to its own file as well, or do you have a better solution?

Reply:
Moving PopResult as well sounds good to me :+1: